src/test/csi-neonsan/templates/snapshot-controller.yaml

```diff
@@ -0,0 +1,107 @@
+{{- if semverCompare ">=1.17" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
Snapshot controller is a public component for all CSI plugin.
If people unintall this plugin, snapshot controller will deleted. It will affect other storage plugin.
You should not put it into qingstor csi helm.
You can take the snapshot controller and crd as a prerequsite.

Reply:
i move it out